pkg/engine/engine.go

```diff
@@ -293,20 +278,15 @@ func NewEngine(ctx context.Context, cfg *Config) (*Engine, error) {
 				return false
 			}
 
-			if !cfg.CustomVerifiersOnly || len(urls) == 0 {
-				customizer.UseFoundEndpoints(true)
-				customizer.UseCloudEndpoint(true)
+			if cfg.CustomVerifiersOnly && len(urls) > 0 {
+				customizer.UseCloudEndpoint(false)
+				customizer.UseFoundEndpoints(false)
```

Comment:
Are we sure we want found endpoints off here?

Reply:
That's a good question. We currently don't have controls to separate the configured cloud endpoint vs found endpoints though, so I defaulted to tying them together.

If a user specifies `--custom-verifiers-only` imo that means "only use the endpoints I specifically configured"